monitoring/prober/README.md

```diff
@@ -98,7 +98,7 @@ new tests.
 ### Creating a new test
 When creating a new test, the first decision is where to locate the test.  The
 prober is divided by subsystems with each subsystem tested in its respective
-folder ([`aux`](aux), [`rid`](rid), [`scd`](scd)).  Each test_*.py file in those
+folder ([`aux`](aux_), [`rid`](rid), [`scd`](scd)).  Each test_*.py file in those
```

Comment:
This is just pointing to the actual folder which has the underscore.  I believe the folder has an underscore because simply `aux` is a disallowed folder/file name in Go.